client/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
Remove default Readme generated by create-react-app and add details about components figma design in Readme.

Reply:
Done